client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-feature-previews/jetpack-preview-pane.tsx

```diff
@@ -43,14 +31,23 @@ export function JetpackPreviewPane( {
 
 	const { selectedSiteFeature, setSelectedSiteFeature } = useContext( SitesDashboardContext );
 
+	useEffect( () => {
+		if ( selectedSiteFeature === undefined ) {
+			setSelectedSiteFeature( 'jetpack_boost' );
+		}
+		return () => {
+			setSelectedSiteFeature( undefined );
+		};
+	}, [] );
+
 	// Jetpack features: Boost, Backup, Monitor, Stats
 	const features = useMemo(
 		() => [
 			createFeaturePreview(
 				'jetpack_boost',
 				'Boost',
 				true,
-				selectedSiteFeature,
+				selectedSiteFeature ?? 'jetpack_boost',
```

Comment:
This indicates that if there are not any selectedSiteFeature, by default, it is jetpack_boost, so the tab knows that it has to mark it as selected.

Reply:
If you use a string for this rather than a constant, you will have to replace it everywhere if the default feature changes. So, let's use a constant.

Reply:
and here @gogdzl